MerlinAU.sh

```diff
@@ -1879,7 +1884,8 @@ Would you like to use the ROG build? (y/n)${NOct}\n"
         -F "firmver=${dottedVersion}" \
         -F "file=@${firmware_file}" \
         --cookie /tmp/cookie.txt > /tmp/upload_response.txt 2>&1 &
-        sleep 60
+        sleep 180
```

Comment:
Extended the sleep timer from 60 to 180 (3 minutes) to be sure the update should be completed before we throw a reboot command.